README.md

```diff
@@ -4,198 +4,25 @@
 [![GitHub issues](https://img.shields.io/github/issues/za419/CadenceBot?style=flat-square)](https://github.com/za419/CadenceBot/issues)
 [![GitHub forks](https://img.shields.io/github/forks/za419/CadenceBot?style=flat-square)](https://github.com/za419/CadenceBot/network)
 
-CadenceBot is a [Discord](https://discordapp.com/) bot, which serves as a client for [Cadence Radio](https://github.com/kenellorando/cadence).
+**CadenceBot** is a [Discord](https://discordapp.com/) bot client for a [Cadence Radio](https://github.com/kenellorando/cadence) stack. 
 
-CadenceBot is fully featured, and is functionally equivalent to the [web client](http://cadenceradio.com/). You can add CadenceBot to your Discord server by clicking [this link.](https://discord.com/api/oauth2/authorize?client_id=372999377569972224&permissions=274881252352&scope=bot)
+It allows you to control your station through a Discord server and even listen to the radio in voice channels! CadenceBot is fully featured and functionally equivalent to the [web client](http://cadenceradio.com/). 
```

Comment:
Should we even go a step further and remove "...fully featured and functionally equivalent to the web client"?

This part of the readme was very much written towards the vision of it as a tech demo when the bot first came into being, with you (@kenellorando) in mind as the reader seeing what's been added - Since we're taking the step of moving it towards being for 'end-users', perhaps that content is no longer required at all.

Reply:
Will update.